command/server/config.go

```diff
@@ -112,6 +112,8 @@ type Config struct {
 
 	DetectDeadlocks string `hcl:"detect_deadlocks"`
 
+	ImpreciseLeaseRoleTracking bool `hcl:"imprecise_lease_role_tracking"`
```

Comment:
Though it's more verbose, how about `imprecise_lease_count_quota_role_tracking`?

I worry that "lease role" won't mean a lot to customers, and changing the name in the future if it's confusing will be difficult.

Reply:
I'm fine with this change. Although, I think documentation would help here the most. A lot of vault config names/env vars names are confusing to me until I read the docs

Reply:
wow I keep going back and forth on this one. The only concern I have is that `imprecise_lease_count_quota_role_tracking` is a bit of a mouthful.  @ncabatoff @mpalmi do you have a preference?

Reply:
If we want to go this route, perhaps `role_tracking` is too much of an implementation detail and we could get away with `skip_pre_quota_lease_tracking`, or `skip_old_lease_counts_on_quota_creation`?

I'm honestly cool with any name we land on. This is a tricky one to pin down and I tend to agree that the documentation is the important part.